gradle/libs.versions.toml

```diff
@@ -1,28 +1,28 @@
 [versions]
-keycloakVersion = "26.0.7"
 amazonSoftwareVersion = "2.29.29"
-junitJupiterVersion = "5.11.3"
 apacheLoggingVersion = "2.24.2"
 bouncyCastleVersion = "1.79"
+junitJupiterVersion = "5.11.3"
+keycloakVersion = "26.0.7"
 
 [libraries]
 comAmazonaws-awsLambdaJavaCore = "com.amazonaws:aws-lambda-java-core:1.2.3"
 comAmazonaws-awsLambdaJavaEvents = "com.amazonaws:aws-lambda-java-events:3.14.0"
+comAmazonaws-awsLambdaJavaLog4j2 = "com.amazonaws:aws-lambda-java-log4j2:1.6.0"
 softwareAmazonAwssdk-cognitoidentityprovider = { group = "software.amazon.awssdk", name = "cognitoidentityprovider", version.ref = "amazonSoftwareVersion" }
-comFasterxmlJacksonDataformat-jacksonDataformatYaml = "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.18.2"
 softwareAmazonAwssdk-ssm = { group = "software.amazon.awssdk", name = "ssm", version.ref = "amazonSoftwareVersion" }
-orgKecloak-keycloakSamlCore = { group = "org.keycloak", name = "keycloak-saml-core", version.ref = "keycloakVersion" }
-orgKecloak-keycloakServices = { group = "org.keycloak", name = "keycloak-services", version.ref = "keycloakVersion" }
-orgBouncycastle-bcprovJdk18on = { group = "org.bouncycastle", name = "bcprov-jdk18on", version.ref = "bouncyCastleVersion" }
-orgBouncycastle-bcpkixJdk18on = { group = "org.bouncycastle", name = "bcpkix-jdk18on", version.ref = "bouncyCastleVersion" }
-orgApacheLoggingLog4j-log4jSlf4j2Impl = { group = "org.apache.logging.log4j", name = "log4j-slf4j2-impl", version.ref = "apacheLoggingVersion" }
```

Comment:
This dependency was removed completely, when running the Lambda, it complained that now there were two different logger implementation classes on the classpath. It seems likely that the CDK code brought in a transitive dependency, and rather than trying to figure out excluding that transitive guy, I just opted to remove this one.

Reply:
You can't tell very easily from the dependency name, but thankfully, I think this dependency (and all others in the repo now) are actually related to `log4j2`, which fixes the security issues in `log4j`.

The original Serverless example code I used to create the Java repo was using Log4j v1 (thanks, Serverless), but I updated to Log4j2 before this went live.

Also, very annoyingly, instead of referring to the new code as `log4j2`, Apache just changed the code path from `org.apache.log4j` for v1 to `org.apache.logging.log4j` for v2 (see [here](https://github.com/newjersey/custom-aws-idp/commit/dfcd98bfc93a1399af501e9dba9fc42649f45ee8#diff-3a51cad948a6df84737801056ddc242cac08e7fd300e07473af8faaf32accf6a)).